test/index.js

```diff
@@ -128,6 +129,106 @@ test("constructs and calls callback without throwing", function (assert) {
     assert.end()
 })
 
+test("[func] xhr[method] get, put, post, patch", function (assert) {
+  var i = 0
+  methods = ["get", "put", "post", "patch"]
+  forEach(methods, function (method) {
+      xhr[method]({
+          uri: "https://httpbin.org/" + method
```

Comment:
hmmm, interesting. IE behaved like it wasn't.

oh, wait. that's probably `XDR` lacking support for methods other than GET&POST.